ansys_optical_automation/application/radiance_from_view.py

```diff
@@ -0,0 +1,32 @@
+from ansys_optical_automation.speos_process.speos_sensors import RadianceSensor
+
+
+def main():
+    Command.Execute("FlyThroughCamera")
```

Comment:
Error "Command" is not defined when running from Ansys Speos, you will need Command back to the base.py

Reply:
i added a fix for it